src/ansi-c/CMakeLists.txt

```diff
@@ -3,20 +3,24 @@ generic_flex(ansi_c)
 
 add_executable(converter library/converter.cpp)
 
+file(GLOB ansi_c_library_sources "library/*.c")
+
 add_custom_command(OUTPUT converter_input.txt
-    COMMAND cat ${CMAKE_CURRENT_SOURCE_DIR}/library/*.c > converter_input.txt
+    COMMAND cat ${ansi_c_library_sources} > converter_input.txt
+    DEPENDS ${ansi_c_library_sources}
 )
 
-add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc
-    COMMAND converter < converter_input.txt > ${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc
-    DEPENDS converter_input.txt
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
+    COMMAND converter < "converter_input.txt" > "${CMAKE_CURRENT_BINARY_DIR}/cprover_library.inc"
+    DEPENDS "converter_input.txt"
```

Comment:
Why are these now wrapped in quotes?

Reply:
It's probably sensible to quote wrap some of those to handle path names with spaces - fairly common on OS X builds, for instance, I suspect.